src/doc/src/reference/publishing.md

```diff
@@ -240,8 +240,23 @@ the “Grant Access” button next to its name:
 
 ![Authentication Access Control](../images/auth-level-acl.png)
 
+#### could not find the github team
+
+Or, when trying to add a GitHub team as crate owner, you may see an error like:
+
+```text
+error: failed to invite owners to crate <crate_name>: api errors (status 200 OK): could not find the github team org/repo
+```
+In that case, you should go to [the GitHub Application settings page] and
+check if crates.io is listed in the `Authorized OAuth Apps` tab.
+If it isn't, you should go to <https://crates.io/login> and authorize it.
+Then go to the crates.io app page, make sure you or your organization
```

Comment:
I'm uncertain what the "app page" is.  

This also might flow a little better like this:

```suggestion
Then go to the crates.io app page and make sure you or your organization
```

Reply:
> I'm uncertain what the "app page" is.

This one: https://github.com/settings/connections/applications/9fe8110dfe185fe90b5c

I'm not sure what's the best wording for it, any idea?

Reply:
Ah, I would try to be explicit about exactly what to do. Maybe something like this:

> Then go back to the Application Settings page on GitHub, click on the
> crates.io application in the list, and make sure you or your organization is
> listed in the "Organization access" list with a green check mark. If there's
> a button labeled `Grant` or `Request`, you should grant the access or
> request the org owner to do so.